crates/hyperqueue/src/worker/parser.rs

```diff
@@ -564,7 +564,7 @@ mod test {
     #[test]
     fn test_parse_resource_def_numeric_value() {
         insta::assert_snapshot!(expect_parser_error(parse_resource_definition_inner(), "x=1"), @r###"
-        Unexpected end of input found while attempting to parse resource kind, expected _ or x:
```

Comment:
This is an error regression :(

Reply:
Do you want to look at? Otherwise I will just revert to the older version.

Reply:
Nah, let's just leave it as it is, the error messages are quite imprecise in other locations too, anyway.

Reply:
Leave it in the updated version version?

Reply:
Yeah let's keep 0.9, it should be faster to compile and includes some nice improvements.